frontend/src/lib/constants.tsx

```diff
@@ -228,7 +228,6 @@ export const UPGRADE_LINK = (cloud?: boolean): { url: string; target?: '_blank'
     cloud ? { url: urls.organizationBilling() } : { url: 'https://posthog.com/pricing', target: '_blank' }
 
 export const DOMAIN_REGEX = /^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$/
-export const SECURE_URL_REGEX = /^(?:http(s)?:\/\/)[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:\/?#[\]@!\$&'\(\)\*\+,;=.]+$/gi
```

Comment:
This is the regex I replaced with a proper URL parser. It accepts both http and https URLS, though I'm not sure if this was intended given the name.